ckanext/switzerland/dcat/harvesters.py

```diff
@@ -90,3 +111,14 @@ def _gen_new_name(self, title):
             return super(SwissDCATRDFHarvester, self)._gen_new_name(title['de'])  # noqa
         except TypeError:
             return super(SwissDCATRDFHarvester, self)._gen_new_name(title)  # noqa
+
+    def before_create(self, harvest_object, dataset_dict, temp_dict):
+        try:
+            source_config_obj = json.loads(harvest_object.job.source.config)
+            if source_config_obj.get('excluded_dataset_identifiers'):
+                for excluded_dataset_identifier in source_config_obj.get('excluded_dataset_identifiers'):  # noqa
+                    if excluded_dataset_identifier == dataset_dict.get('identifier'):  # noqa
+                        dataset_dict.clear()
```

Comment:
Just something small, to "save" one level of indentation, you would generally write this code like this:

```python
for excluded_dataset_identifier in source_config_obj.get('excluded_dataset_identifiers', []):  # noqa
    if excluded_dataset_identifier == dataset_dict.get('identifier'):  # noqa
        dataset_dict.clear()
```

Like that you can get rid of the "if source_config_obj.get('excluded_dataset_identifiers')".

Generally, python follows the "Easier to ask for forgiveness than permission" principle, so if a conditional is not absolutely necessary, try to avoid it.

As I said, it's something small, I will certainly accept the PR without the change as well :wink: